Automattic/jetpack
# Reorg: Fix build artifacts action

Allows Jetpack to build again.

#### Changes proposed in this Pull Request:
*

#### Jetpack product discussion
1199714989172614-as-1199714989172639

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:
* Check "artifacts" and "JS Tests" actions. Make sure they are working fine. 
* Launch a JN site for this branch.
* `cd projects/plugins/jetpack && yarn build`

#### Proposed changelog entry for your changes:


* n/a
